Plot_Data.py

```diff
@@ -0,0 +1,56 @@
+import Use_Data as data
+import matplotlib.pyplot as plt
+from pathlib import Path
+
+def plotdata(toPlot, filename, folder="Main_Sequences_txt_Files", 
+	title="", xtitle="", ytitle="", xlim=[0,0], ylim=[0,0]):
+	"""
+	plotdata takes in an array with the names of columns that are 
+	to be plotted, a filename with the data, and optionally a folder
+	which contains the files as well as many plotting options
+
+
+	Args:
+	    toPlot (np.array): array of names of values to be plotted
+	    filename (str): identifier for file
+	    folder (str): folder name without forward slash
+	"""
+
+	# Get data and header from file
+	arr, header = data.txt2array2D(filepath)
+
+	# Set up for the plotting loop
+	plotlen = len(toPlot)
+	headlen = len(header)
+	exists = False
+	label = -1
+
+	# Plotting loop, starts by checking if toPlot values are valid
+	for i in range(plotlen):
+		for j in range(headlen):
+			if toPlot[i] == header[j]:
+				exists = True
+				label = j
+				break
+
+		if not exists: # If toPlot value doesn't exist tell the user
+			print("There's no", toPlot[i], "in the", filename, "file")
+		
+		else: # Plot data
+			plt.plot(arr[0], arr[i], label=header[label])
+		
+		# Reset to find next toPlot label
+		exists = False
+		label = -1
```

Comment:
Might be a little more readable if the loops grabbed the elements explicitly? 
```suggestion
	for property_data, property_name in zip(arr, header):
		if name in toPlot:
		        plt.plot(arr[0], property, label=name)

		else: # If toPlot value doesn't exist tell the user
			print("There's no", name, "in the", filename, "file")
```
We might need to check that this gives the same results.

Reply:
I changed that up a bit and now it works great